sample-bundle/README.md

```diff
@@ -0,0 +1,16 @@
+# Sample Nomad Bundle 
+
+To use this bundle first you need a nomad cluster up and running. To get a quick one up and running locally, 
+you can run 
+
+```bash
+sudo nomad agent -dev \
```

Comment:
Take out sudo, tell people they may need to run as root dependent on environment, ideally they won't have to though